<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeechCraft</class>
 <widget class="QMainWindow" name="LeechCraft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeechCraft</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainresources.qrc">
    <normaloff>:/resources/images/leechcraft.svg</normaloff>:/resources/images/leechcraft.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="LeechCraft::TabWidget" name="MainTabWidget_">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="MainToolbar_">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionGlance_"/>
   <addaction name="ActionSettings_"/>
  </widget>
  <widget class="QMenuBar" name="MenuBar_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuGeneral_">
    <property name="title">
     <string>General</string>
    </property>
    <addaction name="ActionAddTask_"/>
    <addaction name="ActionNewTab_"/>
    <addaction name="separator"/>
    <addaction name="ActionQuit_"/>
   </widget>
   <widget class="QMenu" name="MenuTools_">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="ActionLogger_"/>
    <addaction name="separator"/>
    <addaction name="ActionSettings_"/>
   </widget>
   <widget class="QMenu" name="MenuHelp_">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ActionAboutLeechCraft_"/>
    <addaction name="ActionAboutQt_"/>
   </widget>
   <widget class="QMenu" name="MenuView_">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="ActionShowStatusBar_"/>
    <addaction name="ActionFullscreenMode_"/>
   </widget>
   <addaction name="MenuGeneral_"/>
   <addaction name="MenuView_"/>
   <addaction name="MenuTools_"/>
   <addaction name="MenuHelp_"/>
  </widget>
  <action name="ActionAddTask_">
   <property name="text">
    <string>Add task</string>
   </property>
  </action>
  <action name="ActionSettings_">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="ActionQuit_">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="ActionIcon" stdset="0">
    <string>exit</string>
   </property>
  </action>
  <action name="ActionFullscreenMode_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen mode</string>
   </property>
   <property name="toolTip">
    <string>Fullscreen mode</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="ActionLogger_">
   <property name="text">
    <string>Log</string>
   </property>
   <property name="toolTip">
    <string>Log</string>
   </property>
  </action>
  <action name="ActionAboutLeechCraft_">
   <property name="text">
    <string>About LeechCraft...</string>
   </property>
  </action>
  <action name="ActionAboutQt_">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="ActionMenu_">
   <property name="text">
    <string>Menu</string>
   </property>
   <property name="ActionIcon" stdset="0">
    <string>menu</string>
   </property>
  </action>
  <action name="ActionShowStatusBar_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show status bar</string>
   </property>
  </action>
  <action name="ActionCloseTab_">
   <property name="text">
    <string>Close tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="ActionGlance_">
   <property name="text">
    <string>Glance...</string>
   </property>
   <property name="toolTip">
    <string>Show a quick overview of tabs</string>
   </property>
  </action>
  <action name="ActionNewTab_">
   <property name="text">
    <string>New tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::TabWidget</class>
   <extends>QTabWidget</extends>
   <header>tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainresources.qrc"/>
 </resources>
 <connections/>
</ui>
